htdocs/contact/class/contact.class.php

```diff
@@ -739,7 +739,7 @@ public function fetch($id, $user = null, $ref_ext = '', $email = '')
 				$this->entity			= $obj->entity;
 				$this->ref				= $obj->rowid;
 				$this->ref_ext			= $obj->ref_ext;
-
+				$this->civility_id    	= $obj->civility_code;
```

Comment:
civility_id must contains the id, not the code of civility.
If id is unknown, property must be kept unknown.